modules/installation-aws-permissions-iam-roles.adoc

```diff
@@ -53,10 +60,24 @@ The control plane and compute machines require the following IAM role permission
 * `kms:DescribeKey`
 ====
 
-.Required IAM role permissions for compute instance profiles
+.Default IAM role permissions for worker instance profiles
 [%collapsible]
 ====
-* `sts:AssumeRole`
 * `ec2:DescribeInstances`
 * `ec2:DescribeRegions`
 ====
+
+= Custom permissions for IAM Instance Profiles
+
+Depending on your cluster set of operators and their configuration, it may be necessary to grant aditional permissions to the instance profiles. Instead of manually adding each permission, it might be more convenient to https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_generate-policy.html?icmpid=docs_iam_console[Generate policies based on access activity].
```

Comment:
```suggestion
Depending on your cluster set of operators and their configuration, it may be necessary to grant additional permissions to the instance profiles. Instead of manually adding each permission, you may https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_generate-policy.html?icmpid=docs_iam_console[generate policies based on access activity].
```